components/molecules/m-update-personal-data.vue

```diff
@@ -76,7 +78,7 @@ export default {
         });
         return;
       }
-      let updatedProfile = JSON.parse(JSON.stringify(this.$store.state.user.current))
+      let updatedProfile = pick(JSON.parse(JSON.stringify(this.$store.state.user.current)), [...config.users.allowModification, 'default_billing'])
```

Comment:
change it to
`let updatedProfile = pick(JSON.parse(JSON.stringify(this.$store.state.user.current)), config.users.allowModification)`
it's fixed in 1.12.3 in core files

Reply:
https://github.com/DivanteLtd/vue-storefront/pull/4929/files#diff-99fb126f4b6264f0d769dab6ec4a3720R92

Reply:
Changed.